akka-persistence-typed/src/main/mima-filters/2.9.6.backwards.excludes/32516-ReplyEffect-thenStop.excludes

```diff
@@ -0,0 +1 @@
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.persistence.typed.scaladsl.ReplyEffect.thenStop")
```

Comment:
mima file should have the latest release in the name, i.e. 2.9.5

Reply:
Weird... one of the commits had it the mima in `2.9.5.backwards.excludes` and mima rejected (https://github.com/akka/akka/pull/32516/commits/07aa5a9212337cb7993a22b4a62fb7dd0dbc1beb)

Reply:
Ah that's it... the first time, it just complained about event-sourced